package.json

```diff
@@ -20,6 +20,7 @@
     "eslint-plugin-jest": "^22.7.2",
     "eslint-plugin-mocha": "^5.3.0",
     "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react-hooks": "^2.2.0",
```

Comment:
You might need to bump the "version" in this file FWIW. Probably doesn't matter though

Reply:
I will bump the version.